config/locales/en.yml

```diff
@@ -248,6 +248,17 @@ en:
       "npq-early-years-leadership": "Early years leadership"
       "npq-leading-literacy": "Leading literacy"
 
+  funding_details:
+    in_review: "Contact your provider to check if you're eligible for scholarship funding, if you have not done so already."
+    ineligible_setting: "You're not eligible for scholarship funding because you do not work in one of the eligible settings, such as state-funded schools."
+    inside_catchment: "You're not eligible for scholarship funding because you do not work in England."
+    no_Ofsted: "You're not eligible for schools funding because you or your employer is not registered on the Ofsted Early Years Register or with a registered Childminder Agency."
```

Comment:
```suggestion
    no_Ofsted: "You’re not eligible for schools funding because you or your employer is not registered on the Ofsted Early Years Register or with a registered Childminder Agency."
```